dev/reformat

```diff
@@ -22,4 +22,4 @@ KYUUBI_HOME="$(cd "`dirname "$0"`/.."; pwd)"
 
 PROFILES="-Pflink-provided,hive-provided,spark-provided,spark-block-cleaner,spark-3.3,spark-3.2,spark-3.1,spark-3.0,tpcds"
 
-${KYUUBI_HOME}/build/mvn spotless:apply $PROFILES
+mvn spotless:apply $PROFILES
```

Comment:
Please revert this change when this PR get ready, the `${KYUUBI_HOME}/build/mvn` will pick up the local `mvn` if there is apposite `mvn` in your PATH rather than download a new one